src/SFA.DAS.EAS.Portal/DependencyResolution/ApplicationServices.cs

```diff
@@ -1,43 +1,23 @@
-﻿using Microsoft.Extensions.Configuration;
-using Microsoft.Extensions.DependencyInjection;
-using SFA.DAS.CommitmentsV2.Messages.Events;
-using SFA.DAS.EAS.Portal.Application.AccountHelper;
-using SFA.DAS.EAS.Portal.Application.Adapters;
+﻿using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Hosting;
 using SFA.DAS.EAS.Portal.Application.Commands;
-using SFA.DAS.EAS.Portal.Application.Commands.Cohort;
-using SFA.DAS.EAS.Portal.Application.Commands.Reservation;
 using SFA.DAS.EAS.Portal.Application.Services;
-using SFA.DAS.EAS.Portal.Configuration;
-using SFA.DAS.HashingService;
 
 namespace SFA.DAS.EAS.Portal.DependencyResolution
 {
     public static class ApplicationServices
     {
-        public static IServiceCollection AddApplicationServices(this IServiceCollection services)
+        public static IServiceCollection AddApplicationServices(this IServiceCollection services, HostBuilderContext hostBuilderContext)
         {
-            services.AddTransient<AddReservationCommand>();
-            var serviceProvider = services.BuildServiceProvider();
-            var configuration = serviceProvider.GetService<IConfiguration>();
-
-            services.AddCommitmentsApiConfiguration(configuration);
-
-            // hashing service config            
-            var hashServiceConfig = configuration.GetPortalSection<HashingServiceConfiguration>(PortalSections.HashingService);            
-            services.AddSingleton<IHashingService>(s => new HashingService.HashingService(hashServiceConfig.AccountLegalEntityPublicAllowedCharacters, hashServiceConfig.AccountLegalEntityPublicHashstring));
-            
             services.AddScoped<IMessageContext, MessageContext>();            
             services.AddTransient<IAccountDocumentService, AccountDocumentService>();                        
             services.Decorate<IAccountDocumentService, AccountDocumentServiceWithSetProperties>();
             services.Decorate<IAccountDocumentService, AccountDocumentServiceWithDuplicateCheck>();
-            services.AddScoped<IAccountHelperService, AccountHelperService>();
-            services.AddTransient<IAdapter<CohortApprovalRequestedByProvider, CohortApprovalRequestedCommand>, CohortAdapter>();
 
-            // Register all ICommandHandler<> types
             services.Scan(scan =>
-                    scan.FromAssembliesOf(typeof(ICommandHandler<>))
+                    scan.FromAssembliesOf(typeof(ICommand<>))
```

Comment:
If it looks like a Handler and walks like a Handler it is a Handler.  Please rename back to ICommandHandler.

Reply:
Funny! 😂

If it was an interface for a command handler, then it would be appropriate to call the interface `ICommandHandler`. As it's an interface for a command, then `ICommand` is appropriate.

Once we decide on the Command vs Command+Command Handler pattern, we can name the interfaces appropriately.